services/QuillLMS/client/app/bundles/Connect/components/questions/responsesManager.jsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
```

Comment:
Are you planning to consolidate the three responseManagers at a later date?

Reply:
In theory, but I think we'd want to do that after moving actions files into a Shared bundle, and we're not there yet. Since the ResponseManagers are each used in several files in their respective bundles, we'd need to couple a lot of state to the parent components in order to just pass this stuff in as props there, and that feels like an antipattern to me.